connectors/resteasy/pom.xml

```diff
@@ -19,6 +19,12 @@
 			<groupId>org.jboss.resteasy</groupId>
 			<artifactId>resteasy-jaxrs</artifactId>
 			<version>${resteasy-version}</version>
+			<exclusions>
+				<exclusion>
+					<artifactId>httpclient</artifactId>
+					<groupId>org.apache.httpcomponents</groupId>
+				</exclusion>
+			</exclusions>
```

Comment:
Why excluding the httpclient from resteasy?

Reply:
It is. Was a little bit ugly to get this running. You need to to this via maven surefire and attach to this. Search for `maven remote debugging`.

Reply:
Were you able to progress here @kushalagrawal1? Else we could merge as-is and release.
Could still go for the resteasy upgrade afterwards.
Ping me if you need help

Reply:
Hi @auhlig, tried setting up IntelliJ but couldn't get succeed. got stuck in regular office tasks.
I can send you git patch if you are ok. I will try to get some time this weekend and finish the task otherwise.

Reply:
No worries. I'm also happy to take this in it's current state. Just wanted to know, so do not postpone the release unnecessarily. Many thanks for all the work!
So is it good to merge from your side @kushalagrawal1?

Reply:
it works for me

Reply:
@aiziyuer  can you please apply this patch and try?
<change the extension of the file to ".patch">
https://github.com/ContainX/openstack4j/files/1291381/issue_1070_patch.txt

Reply:
Hi @auhlig @vinodborole, I have done all required changes from my side. can you please review it once so it can be merged, or if there is any change need to be done will not affect release.